spec/vast_trackers.spec.js

```diff
@@ -282,5 +285,41 @@ describe('VASTTracker', function() {
         expect(spyTrack).not.toHaveBeenCalledTimes(2);
       });
     });
+    describe('#error', () => {
+      let isCustomCode = false;
+      beforeEach(() => {
+        vastTracker.error();
+      });
+      it('should be defined', () => {
+        expect(ad.errorURLTemplates).toBeDefined();
```

Comment:
You don't need to check if `errorURLTemplates` is defined. The check that you did l.297 is enough 😄